### DISCUSSION TITLE
import personnal phantom

### DISCUSSION THREAD

**Discussion by FasicolZyy** [e0404/matRad#722] [Q&A]
I would like to ask how to import my own body phantom in Matrad. I have learned that there are public datasets such as TG119 model and liver model in Matrad. What should I do to import my own model? My model is not in dicom format, but a point cloud file.

**Comment by wahln** [comment]
I have no experience with point cloud files, and matRad does not support an input. matRad uses voxel geometries for planning, so I don't know how this would be imported in the best way. Maybe an image processing platform like Slicer or MITK can import point-cloud-files and export them to DICOM or binary voxel images / masks.
